src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -350,10 +349,11 @@ case class CoveringIndex(properties: CoveringIndex.Properties) {
   val kindAbbr = "CI"
 }
 object CoveringIndex {
-  case class Properties(columns: Properties.Columns,
-    schemaString: String,
-    numBuckets: Int,
-    properties: Map[String, String])
+  case class Properties(
```

Comment:
Same here. No changes required. Here is how this part looks after I applied `scalafmt` reformat command:

```
object CoveringIndex {
  case class Properties(columns: Properties.Columns,
    schemaString: String,
    numBuckets: Int,
    properties: Map[String, String])

  object Properties {
    case class Columns(indexed: Seq[String], included: Seq[String])
  }
}
```

No changes.